flang/include/flang/Lower/FIRBuilder.h

```diff
@@ -69,8 +70,20 @@ class FirOpBuilder : public mlir::OpBuilder {
   mlir::Value convertWithSemantics(mlir::Location loc, mlir::Type toTy,
                                    mlir::Value val);
 
-  /// Get the entry block of the current Function
-  mlir::Block *getEntryBlock() { return &getFunction().front(); }
```

Comment:
We might want to leave `getEntryBlock()` as it reads clearer than `.front()` to me.

Reply:
Done.